doc: write tcm release notes (#5584)
tarantool/doc: doc/tooling/tcm/tcm_releases/tcm_1.6.rst

TDB workers monitoring in cluster dashboard
--------------------------------------------

|tcm| adds support for TDB workers in the cluster dashboard with integrated health monitoring and visibility.

Workers are automatically discovered from etcd and continuously monitored via dedicated health check endpoints.
Their metrics are proxied through TCM and exposed individually, allowing detailed operational insight.

The interface displays workers directly in the stateboard with clear status indicators and a details panel.
Each worker can be in one of four states: healthy, degraded, unhealthy, or no connection, helping administrators quickly detect and diagnose issues.

.. _tcm_releases_1_6_audit_log:

Audit log configuration improvements
------------------------------------

The audit log configuration is now safer and more predictable.

Protocol values are validated during startup. If an invalid protocol is specified, the system automatically falls back to default settings and emits a warning.
Audit log parameters can also be preconfigured during system bootstrap through the initial configuration file and are automatically applied if audit logging has not yet been set up.

> 

Reviewer: ```suggestion
Audit log parameters can be set in advance at the system bootstrap stage by specifying them in the auditlog field of the initial-settings section in the configuration file. These settings will be applied automatically if the audit log has not been configured yet.

```

.. _tcm_releases_1_6_explorer:

Explorer enhancements
---------------------

A feature flag has been introduced to control the visibility of the **Tuples** tab in the Explorer interface.

The tab is displayed only when the corresponding feature flag is enabled and the CRUD module is available.
The flag can be configured either in the application configuration file or via command-line arguments at startup

.. _tcm_releases_1_6_fixes:

Stability and reliability fixes
-------------------------------

This release also includes several fixes that improve system stability and security.

> LDAP authentication behavior has been corrected, including logout handling, anonymous binding to Active Directory, and preservation of authorization settings after restart.

Reviewer: ```suggestion

LDAP authentication behavior has been adjusted, including logout handling, anonymous binding to Active Directory, and the preservation of authorization method settings after a restart.
```

.. _tcm_releases_1_6:

Tarantool Cluster Manager 1.6
=============================

Release date: February 6, 2026

Latest release in series: 1.6.0

> This release introduces support for TDB workers in the cluster dashboard with integrated health monitoring,

Reviewer 2: Я бы написала тут полностью первое вхождение названия
Tarantool DataBase (TDB)

adds TLS configuration guides for secure connections, improves audit log configuration and validation,
and introduces a feature flag for managing the Tuples tab. It also includes important fixes for LDAP authentication,

> TLS configuration parsing, and a memory leak in SSL cluster connections.

Reviewer 3: фиксы 1.6 ниже перечислены более презентабельно, каждый фикс на отдельной строчке, может и тут так сделать?

Author: @AArdeev где тут?

.. _tcm_releases_1_6_workers_tdb:

TDB workers monitoring in cluster dashboard
--------------------------------------------

> |tcm| adds support for TDB workers in the cluster dashboard with integrated health monitoring and visibility.

Reviewer 2: 1) Надо добавить ссылку на раздел про воркеры:
https://www.tarantool.io/ru/tarantooldb/doc/latest/examples/tdb_worker/tdb_worker_example/
2) нужно написать, что воркеры поддерживаются с версии Tarantool DB 3.1.0.

> |tcm| adds support for TDB workers in the cluster dashboard with integrated health monitoring and visibility.

Reviewer 2: Мы вроде используем нейминг stateboard, и вкладка так тоже называется 
Я бы дописала еще, что воркеры отображаются на вкладке **Stateboard**

> Workers are automatically discovered from etcd and continuously monitored via dedicated health check endpoints.

Reviewer 3: discovered from etcd and /then/ continuously monitored
я бы добавил эту связку

Author: а что тут не так?

Their metrics are proxied through TCM and exposed individually, allowing detailed operational insight.

The interface displays workers directly in the stateboard with clear status indicators and a details panel.

> Each worker can be in one of four states: healthy, degraded, unhealthy, or no connection, helping administrators quickly detect and diagnose issues.

Reviewer 3: Если речь в 1м предложении и во 2м об одном и том же определении состояния, то предлагаю назвать его одинаково - state или status.

.. _tcm_releases_1_6_audit_log:

Audit log configuration improvements
------------------------------------

The audit log configuration is now safer and more predictable.

Protocol values are validated during startup. If an invalid protocol is specified, the system automatically falls back to default settings and emits a warning.

> Audit log parameters can be set in advance at the system bootstrap stage by specifying them in the ``auditlog`` field of the initial-settings section in the configuration file.

Reviewer 3: in an advance кажется

Author: нет, такое ревью перебор уже)

These settings will be applied automatically if the audit log has not been configured yet.

.. _tcm_releases_1_6_explorer:

Explorer enhancements
---------------------

A feature flag has been introduced to control the visibility of the **Tuples** tab in the Explorer interface.

The tab is displayed only when the corresponding feature flag is enabled and the CRUD module is available.

> The flag can be configured either in the application configuration file or via command-line arguments at startup

Reviewer 2: Тут просится код-блок с настройкой этой фичи

> The flag can be configured either in the application configuration file or via command-line arguments at startup

Reviewer 2: Давай напишем в кодблоке, как включить в конфиге TCM опцию:

```yaml
feature:
  tuples: True
```

PS: меня слегка запутала формулировка application configuration file - у меня сложилось впечатление, что речь идет про приложение в смысле кластер. Я бы так явно и сформулировала, что в конфигурации TCM можно задть

.. _tcm_releases_1_6:

Tarantool Cluster Manager 1.6
=============================

Release date: February 6, 2026

Latest release in series: 1.6.0

This release introduces support for Tarantool DataBase (TDB) workers in the cluster dashboard with integrated health monitoring,
adds TLS configuration guides for secure connections, improves audit log configuration and validation,

> and introduces a feature flag for managing the Tuples tab. It also includes important fixes for LDAP authentication,

Reviewer 2: ```suggestion
and introduces a feature flag for managing the **Tuples** tab. It also includes important fixes for LDAP authentication,
```

--------------------------------------------

|tcm| adds support for TDB workers in the cluster **Stateboard** tab with integrated health monitoring and visibility.

TDB workers are supported starting from TDB 3.1.0.
Workers are automatically discovered from etcd and continuously monitored via dedicated health check endpoints.
Their metrics are proxied through TCM and exposed individually, allowing detailed operational insight.

The interface displays workers directly in the stateboard with clear status indicators and a details panel.
Each worker can be in one of four statuses: healthy, degraded, unhealthy, or no connection, helping administrators quickly detect and diagnose issues.

To learn more, see `TDB documentation <https://www.tarantool.io/ru/tarantooldb/doc/latest/examples/tdb_worker/tdb_worker_example/>`__.

.. _tcm_releases_1_6_audit_log:

Audit log configuration improvements
------------------------------------

The audit log configuration is now safer and more predictable.

Protocol values are validated during startup. If an invalid protocol is specified, the system automatically falls back to default settings and emits a warning.

> Audit log parameters can be set in advance at the system bootstrap stage by specifying them in the ``auditlog`` field of the initial-settings section in the configuration file.

Reviewer 2: ```suggestion
Audit log parameters can be set in advance at the system bootstrap stage by specifying them in the ``auditlog`` field of the ``initial-settings`` section in the configuration file.
```
